doc/src/modules/polys/index.rst

```diff
@@ -1,7 +1,7 @@
 .. _polys-docs:
 
 ===============================
-Polynomials Manipulation Module
+Polynomials Manipulation
 ===============================
```

Comment:
The module is called "polys" but if I were writing a title like above I would use the singular "polynomial manipulation".